index.js

```diff
@@ -339,14 +339,14 @@ function ensureValidOptions(options) {
     if(!options) throw new Error("options are required by express-winston middleware");
     if(!((options.transports && (options.transports.length > 0)) || options.winstonInstance))
         throw new Error("transports or a winstonInstance are required by express-winston middleware");
+
+    if (options.dynamicMeta && !_.isFunction(options.dynamicMeta)) {
+        throw new Error("`dynamicMeta` express-winston option should be a function");
+    }
 }
 
 function ensureValidLoggerOptions(options) {
     if (options.ignoreRoute && !_.isFunction(options.ignoreRoute)) {
         throw new Error("`ignoreRoute` express-winston option should be a function");
     }
-
```

Comment:
Since the option to handle `dynamicMeta` in the `errorLogger` function was added on commit [9ea0300ddd](https://github.com/TuneOSS/express-winston/commit/9ea0300ddd65e28e3d4c3a120baf10cd8a407f6f) , this `dynamicMeta` validation was moved to the `ensureValidOptions` function. That way there is no need to call `ensureValidLoggerOptions` on the `errorLogger` function